src/App.jsx

```diff
@@ -1,12 +1,9 @@
 import React from 'react';
 import { hot } from 'react-hot-loader';
-import { HashRouter, Route, Redirect } from 'react-router-dom';
+import { HashRouter } from 'react-router-dom';
```

Comment:
CRA does polyfill history. I saw when i ejected. If we were to change to a different router, i say that we go all out and use connected-router. That way our routes and everything dealing with it is connected to the store instead of the routes being their own thing.